controller/information_source/manager.py

```diff
@@ -8,12 +8,16 @@ def get_information_source(project_id: str, source_id: str) -> InformationSource
     return information_source.get(project_id, source_id)
 
 
+def get_information_source_by_name(project_id: str, name: str) -> InformationSource:
+    return information_source.get_by_name(project_id, name)
+
+
 def get_all_information_sources(project_id: str) -> List[InformationSource]:
     return information_source.get_all(project_id)
 
 
-def get_overview_data(project_id: str) -> str:
-    return information_source.get_overview_data(project_id)
+def get_overview_data(project_id: str, operator: Optional[str] = "!=") -> str:
```

Comment:
optional is only for = None 
[notion link](https://www.notion.so/kern-ai/Coding-Guide-Python-f54c6201aa22471caaff38d765c0d64e#447b4b18e8b94725b8ad0c767445fcb3)

Reply:
also, im not the biggest fan of operator as variable since it's pretty undescriptive what it's for.

What do you think about `,for_model_callback: bool = false)->str` and in the information source function something like
``` python
type_selection = " != 'MODEL_CALLBACK'"
if for_model_callback:
     type_selection = " = 'MODEL_CALLBACK'"
```

same for update_is_selected_for_project

Reply:
makes sense, changed that